modules/ztp-adding-new-content-to-gitops-ztp.adoc

```diff
@@ -3,7 +3,109 @@
 // * scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.adoc
 
 :_content-type: PROCEDURE
+
+###################################################
 [id="ztp-adding-new-content-to-gitops-ztp_{context}"]
+= Adding custom content to the {ztp} pipeline
```

Comment:
Instead of adding this procedure to the `modules/ztp-adding-new-content-to-gitops-ztp.adoc` module, suggest creating a new module, `modules/ztp-adding-custom-crs-to-gitops-ztp.adoc`

Reply:
The doc story suggested that the work done in [CNF-6925](https://issues.redhat.com/browse/CNF-6925) likely obsoletes the procedure in [Adding new content to the GitOps ZTP pipeline](https://docs.openshift.com/container-platform/4.11/scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.html#ztp-adding-new-content-to-gitops-ztp_ztp-advanced-policy-config)

The suggestion was to rewrite the existing module, preserving the file name/id for backwards compatibility. 
SME supported that suggestion so I've gone with that.